src/app/child-dev-project/reporting/reporting/reporting.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit } from "@angular/core";
+import { ActivatedRoute } from "@angular/router";
+import { Aggregation, ReportingService, ReportRow } from "../reporting.service";
+
+export interface ReportingComponentConfig {
+  aggregationDefinitions?: Aggregation[];
+}
+
+@Component({
+  selector: "app-reporting",
+  templateUrl: "./reporting.component.html",
+  styleUrls: ["./reporting.component.scss"],
+})
+export class ReportingComponent implements OnInit {
+  config: ReportingComponentConfig;
+  results: ReportRow[];
+  fromDate: Date;
+  toDate: Date;
+  reportTable: { label: string; result: any }[];
```

Comment:
this is only used for export? rename to `resultsForExport` or something that is more clear?